### DISCUSSION TITLE
直接在SFC中导入原组件，报错Hydration completed but contains mismatches.

### DISCUSSION THREAD

**Discussion by Beny2305** [vuepress-theme-hope/vuepress-theme-hope#3300] [Q&A] [answered]
比如新建一个组件 HbNavbar.vue , 然后copy原组件Navbar所有内容到<script>标签下，修正scss路径，替换以后可以正常运行， 但build版本出现错误: Hydration completed but contains mismatches. 请问这是什么问题导致的呢，（直接导入到js文件替换没有问题）

**Comment by Mister-Hope** [comment, marked as answer]
reproduction,别人很难那从文字理解你的潜在错误。

主题文档本身就进行了组件替换，功能肯定是没问题的。
